docs/source/properties_computed.rst

```diff
@@ -0,0 +1,113 @@
+Computed Properties
+===================
+
+Helanal follows the procedure of Sugeta and Miyazawa [Sugeta1967]_. Properties
+are computed for a 'window' of four consecutive :math:`C_α` atoms, and this 
+window then slid along the length of the helix in one-residue steps.
+
+For each window consisting of atoms :math:`c_i`, :math:`c_{i+1}`, 
+:math:`c_{i+2}`, :math:`c_{i+3}`, the vectors :math:`\mathbf{B_1}`, 
+:math:`\mathbf{B_2}`, and :math:`\mathbf{B_3}` joining (respectively) atoms 
+:math:`c_i`--:math:`c_{i+1}`, :math:`c_{i+1}`--:math:`c_{i+2}` and 
+:math:`c_{i+2}`--:math:`c_{i+3}` are calculated, along with the vectors 
+:math:`\mathbf{D_1} = \mathbf{C_1} - \mathbf{C_2}` and 
+:math:`\mathbf{D_2} = \mathbf{C_2} - \mathbf{C_3}`. 
+
+.. image:: images/window.svg
+   :width: 60%
+   :align: center
+
+From these, the helix properties below are computed in each simulation frame.
+These properties are available in ``.results`` as arrays, the shape of which
+depends on the number of residues :math:`n_{res}` (and the property being 
+calculated). **Note that each helix must contain at least 9 residues.**
+
+If multiple helices are being analysed, the results are returned as lists (of
+length :math:`n_{helices}`) of arrays of the indicated shape.
+
+All angles are in degrees.
+
+.. list-table::
+   :widths: 40 30 20
+   :header-rows: 1
+
+   * - Description
+     -  
+     - Shape
+   * - ``local_helix_directions``: the normalised vector :math:`\mathbf{D_1}` 
+       (or :math:`\mathbf{D_2}`) for each atom :math:`c_{i+1}` (or 
+       :math:`c_{i+2}`).
+
+       Assuming ~even spacing of the atoms, this vector will bisect the angle 
```

Comment:
I'm not sure the "Assuming ~even spacing" would be helpful wording to me as a user trying to figure out what this is, although I have a serious case of brain-fuzz at the moment. Or maybe if there was an additional footnote explaining why it's only approximately bisecting it because the B_x vectors aren't normalised would be helpful.

Reply:
Very fair - I've sort of addressed this above.

As said above, I don't *think* the fact these are bisectors is actually specifically relevant. I'd kept the references because the code is still strongly linked to it (e.g. with the variable for the D vectors being called `bisectors`) and I didn't want to mess with those, and thought it might help people visualise what's going on - but with all my cautious it's only approximate it may be adding confusion more than helping. 

Perhaps for now, I could simplify to only saying they'll point from the local origin to the atom, and maybe in the future add some notes later expanding on the exact calculation, something like the above?

Reply:
Maybe saying both would be clearest? So something like "this vector points from the local origin to the atom. Assuming ~even spacing of the atoms, ..."

> some notes later expanding on the exact calculation

This would be amazing so we don't keep revisiting this 😅 And for any user who *really really* wants to know what's happening without reading the paper or the code...